engine/transaction/draft/draft_service.go

```diff
@@ -13,19 +14,25 @@ import (
 )
 
 type service struct {
-	logger         *zerolog.Logger
-	paymailService paymail.ServiceClient
+	logger                *zerolog.Logger
+	paymailService        paymail.ServiceClient
+	paymailAddressService paymailaddress.Service
 }
 
 // NewDraftService creates a new draft service.
-func NewDraftService(paymailService paymail.ServiceClient, logger zerolog.Logger) Service {
+func NewDraftService(paymailService paymail.ServiceClient, paymailAddressService paymailaddress.Service, logger zerolog.Logger) Service {
 	if paymailService == nil {
-		panic("paymail service is required to create draft transaction service")
+		panic("paymail.ServiceClient is required to create draft transaction service")
+	}
+
+	if paymailAddressService == nil {
+		panic("paymailaddress.Service is required to create draft transaction service")
```

Comment:
I'm not sure if the panics should be here.
This function is used in `loadTransactionDraftService` which could return some error on fail.
For now it returns always `nil`.

Reply:
Actually after rethinking previous approaches, I think that actually we should always panic in such case in NewXyz()
It shouldn't happen at all, and if set in wrong way, than it is 100% fault of programmer and he need to fix it as soon as possible.

I would like even change all the loadXyz functions in client_internal to panic but I think that it is again just a tip of iceberg.

Reply:
So ... maybe we should remove an option to return an error from `loadTransactionDraftService` - and perhaps for siilars 🤔 ?

Reply:
maybe in the future 🙈 
Or I will try to do that just after PR with custom assertions.